docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+services:
+
+  web:
+    build:
+      dockerfile: Dockerfile
+      target: web
+    volumes:
+      - ./config.json:/var/www/reon/config.json:ro
+    expose:
+      - 9000
+
+  nginx:
+    image: nginx:1.27.0
+    volumes:
+      - ./examples/nginx/default.conf.template:/etc/nginx/templates/default.conf.template
+      - ./web/:/var/www/reon/web
+    ports:
+      - 80:80
+    environment:
+      - NGINX_PORT=80
+      - NGINX_HOST="localhost gameboy.datacenter.ne.jp"
+      # Hostname and port for fastcgi_pass directive
+      - WEB_URL=web:9000
+
+  mail:
+    build:
+      dockerfile: Dockerfile
+      target: mail
+    ports:
+      - 25:25
+      - 110:110
+    volumes:
+      - ./config.json:/home/node/config.json:ro
+
+  db:
+    image: mysql:8.4.0
+    ports:
+      - 3306:3306
+    volumes:
+      # Run tables.sql on startup
+      - ./tables.sql:/docker-entrypoint-initdb.d/tables.sql:ro
+      - type: volume
+        source: reon_db
+        target: /var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
+      - MYSQL_USER=reon
+      - MYSQL_PASSWORD=password
+      - MYSQL_DATABASE=db
```

Comment:
Working on something to move these out. Most docker setups use a `.env` file, I'd like to look into using that and possibly creating a `config.json` from the values in `.env` or removing `config.json`. Ideally there's just one configuration file.